pkgs/desktops/gnome-3/core/gnome-keyring/default.nix

```diff
@@ -46,6 +46,16 @@ stdenv.mkDerivation rec {
       make check
   '';
 
+  # Use wrapped gnome-keyring-daemon with cap_ipc_lock=ep
+  postFixup = ''
+    files=($out/etc/xdg/autostart/* $out/share/dbus-1/services/*)
+
+    for file in ''${files[*]}; do
+      substituteInPlace $file \
+        --replace "$out/bin/gnome-keyring-daemon" "/run/wrappers/bin/gnome-keyring-daemon"
```

Comment:
Won't this cause issues when gnome-keyring is installed when the system has
```
services.gnome3.gnome-keyring.enable = false;
```
?

Reply:
Or the system is not NixOS…

Reply:
This is a daemon, any usage other than through NixOS module is not supported.

Reply:
Yep, something similar is even done for the dbus-daemon apparently:
https://github.com/NixOS/nixpkgs/blob/73debdc298aeb28fe1a86a0b230b34cb11ea5d93/pkgs/development/libraries/dbus/default.nix#L33